src/components/App.js

```diff
@@ -19,9 +19,15 @@ const lines = [greenLine, orangeLine, redLine];
 const App = () => {
     const api = useMbtaApi(lines);
     const tabState = useTabState({ loop: false });
-    const tabIndex = tabState.currentId
-        ? tabState.items.findIndex(i => i.id === tabState.currentId)
-        : 0;
+    // tabState.items is returning an empty array
```

Comment:
Since we can't use `tabState.items` anymore, could we consider getting rid of `tabIndex` altogether and converting the `lines` array to a `lineByTabId` object that we can index into with the `currentId` directly?

(Also, once you've fixed it you can probably remove this comment — future readers will be wondering what `tabState.items` is since it's no longer referenced anywhere).